docs/usermanual.md

```diff
@@ -348,7 +348,7 @@ to the Context Broker regarding an entity called `sen1` of type `sensor`:
 ```
 
 If the API key associated to de device is `ABCDEF`, and the device ID related to `sen1` entity is `id_sen1`, this will
-generate a message in the `/ABCDEF/id_sen1/cmd` topic with the following payload:
+generate a message in the `/json/ABCDEF/id_sen1/cmd` topic with the following payload:
```

Comment:
Usage prefix should be reviewed in documentation after recent discussions in skype... Originally we thought in using prefix for commands but now we don't use it but not sure if documentation changes have been updated accordinly.

I guess it may affect to .../cmd topic, .../cmdexe topic or both (not sure...).